frame/beefy-mmr/primitives/src/lib.rs

```diff
@@ -375,6 +377,21 @@ where
 	}
 }
 
+sp_api::decl_runtime_apis! {
+	/// API useful for BEEFY light clients.
+	pub trait BeefyMmrApi<H>
+	where
+		H: From<Hash> + Into<Hash>,
+		BeefyAuthoritySet<H>: sp_api::Decode,
+	{
+		/// Return the currently active BEEFY authority set.
+		fn authority_set() -> BeefyAuthoritySet<H>;
+
+		/// Return the next/queued BEEFY authority set.
+		fn next_authority_set() -> BeefyNextAuthoritySet<H>;
```

Comment:
We should probably come up with better names for these in the future as otherwise they might seem very similar to the existing `validator_set` runtime API that exists for Beefy pallet.